de.metas.swat/de.metas.swat.base/src/main/java/de/metas/impex/api/IInputDataSourceDAO.java

```diff
@@ -24,14 +26,21 @@
 
 import java.util.Properties;
 
+import de.metas.impex.InputDataSourceId;
+import de.metas.impex.api.impl.InputDataSourceQuery;
 import de.metas.impex.model.I_AD_InputDataSource;
 import de.metas.util.ISingletonService;
 
 public interface IInputDataSourceDAO extends ISingletonService
 {
+	I_AD_InputDataSource getById(final InputDataSourceId id);
+
 	I_AD_InputDataSource retrieveInputDataSource(Properties ctx, String internalName, boolean throwEx, String trxName);
-	
-	int retrieveInputDataSourceId(String internalName);
+
+	int retrieveInputDataSourceIdByInternalName(String internalName);
```

Comment:
please change the returnvalue to InputDataSourceId

Reply:
Done